docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -263,13 +263,23 @@ public class SubjectExposingResource {
         String name = user != null ? user.getName() : "anonymous";
         return name;
     }
+
+    @GET
+    @Path("config-property-expressions")
+    @RolesAllowed("${admin}") <6>
+    public String getSubjectSecuredConfigExpression(@Context SecurityContext sec) {
+        Principal user = sec.getUserPrincipal();
+        String name = user != null ? user.getName() : "anonymous";
+        return name;
+    }
 }
 ----
 <1> This `/subject/secured` endpoint requires an authenticated user that has been granted the role "Tester" through the use of the `@RolesAllowed("Tester")` annotation.
 <2> The endpoint obtains the user principal from the JAX-RS `SecurityContext`. This will be non-null for a secured endpoint.
 <3> The `/subject/unsecured` endpoint allows for unauthenticated access by specifying the `@PermitAll` annotation.
 <4> This call to obtain the user principal will return null if the caller is unauthenticated, non-null if the caller is authenticated.
 <5> The `/subject/denied` endpoint disallows any access regardless of whether the call is authenticated by specifying the `@DenyAll` annotation.
+<6> The `@RolesAllowed` annotation value supports <<config-reference#property-expressions,Property Expressions>> including default values and nested Property Expressions.
```

Comment:
```suggestion
<6> The `@RolesAllowed` annotation value is set to the value of the `admin` configuration property, see <<config-reference#property-expressions,Property Expressions>> for more information.
```

Reply:
I believe in this example, it has to be a concrete piece of info, can make sense IMHO to have a separate dedicated section with a  more technical overview

Reply:
I agree with dedicated section, just wan't sure it's important enough. I'll write something.

Reply:
I completely re-wrote that, let see please and thank you.